topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -194,6 +197,62 @@ Next, we will compare the short reads to the assembly, and create a polished (co
 * Run a second round (or more) of Pilon polishing. Keep track of file naming; you will need to generate a new bam file first before each round of Pilon.
 * Run an alternative polishing tool, such as Racon. This uses the long reads themselves to correct the long-read (Flye) assembly. It would be better to run this tool on the Flye assembly before running Pilon, rather than after Pilon.
 
+# Annotate the assembly
+
+We can now annotate our assembled genome with information about genomic features.
+
+* A chloroplast genome annotation tool is not yet available in Galaxy; for an approximation, here we can use the tool for bacterial genome annotation, Prokka. 
+
+> ### {% icon hands_on %} Hands-on: Annotate with Prokka
+>
+> 1. {% tool [Prokka](toolshed.g2.bx.psu.edu/repos/crs4/prokka/prokka/1.14.5+galaxy0) %} with the following parameters (leave everything else unchanged)
+>    - {% icon param-file %} *"contigs to annotate"*: `polished-assembly.fasta`
+> 2. **View output**:
+>    * The GFF and GBK files contain all of the information about the features annotated (in different formats.)
+>    * The .txt file contains a summary of the number of features annotated.
+>    * The .faa file contains the protein sequences of the genes annotated.
+>    * The .ffn file contains the nucleotide sequences of the genes annotated.
+{: .hands_on}
+
+
+Alternatively, you might want to use a web-based tool designed for chloroplast genomes. 
+
+* One option is the GeSeq tool, described here. Skip this step if you have already used Prokka above. 
+
+> ### {% icon hands_on %} Hands-on: Annotate with GeSeq
+> * Download `polished-assembly.fasta` to your computer (click on the file in your history; then click on the disk icon).
+> * In a new broswer tab, go to [Chlorobox](https://chlorobox.mpimp-golm.mpg.de/geseq.html) where we will use the GeSeq tool {% cite Tillich_2017 %} to annotate our sequence.
+> * Upload the `fasta` file there. Information about how to use the tool is available on the page.
+> * Once the annotation is completed, download the required files.
+> * In Galaxy, import the annotation `GFF3` file.
+{: .hands_on}
+
+Now make a JBrowse file to view the annotations (the GFF3 file - produced from either Prokka or GeSeq) under the assembly (the polished-assembly.fasta file).
+
+> ### {% icon hands_on %} Hands-on: View annotations
+> 1. {% tool [JBrowse genome browser](toolshed.g2.bx.psu.edu/repos/iuc/jbrowse/jbrowse/1.16.4+galaxy3) %}:
+>        - *"Reference genome to display"*: `Use a genome from history`
```

Comment:
the indentation here looks a bit odd? Should this be with fewer spaces? And "select a reference" below indented below this element

Reply:
Thanks for spotting this! will fix